code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm

```diff
@@ -94,13 +89,14 @@
 /datum/gear/ears/headlights
 	display_name = "headlights"
 	path = /obj/item/device/flashlight/headlights
-	cost = 2
```

Comment:
These provide a very clear utility as a flashlight on top of being a cosmetic.

Reply:
Yeah I actually just undid this